package.json

```diff
@@ -63,6 +63,7 @@
     "update-notifier": "5.1.0"
   },
   "devDependencies": {
+    "@jest/globals": "29.3.1",
```

Comment:
It seems like we are using this as a production dependency now (production is everything that is required to run the codemod, devDependencies is everything required to develop jest-codemods).

Also we need to update the lock file.

Reply:
> It seems like we are using this as a production dependency now (production is everything that is required to run the codemod, devDependencies is everything required to develop jest-codemods).

Ah; fair enough.  I think whether this is a dev dependency, prod dependency, or dependency at all is contingent on the outcome of whether we parse files (i.e. https://github.com/skovhus/jest-codemods/pull/508#discussion_r1401088600)

> 
> Also we need to update the lock file.

I had tried this (running `pnpm install`) but I wasn't seeing any difference in the lock file.  I believe it's because the exact same version is already there(?).  I don't use pnpm much so let me know if I'm messing something up!